idunn/datasources/hove/models.py

```diff
@@ -0,0 +1,338 @@
+"""
+Derialize Hove's responses and conversion primitives following the following equivalence:
```

Comment:
following the following :p (maybe following the next equivalences)

Reply:
:vomiting_face: